ops/op2/signature.rs

```diff
@@ -1469,6 +1481,35 @@ fn parse_cppgc(position: Position, ty: &Type) -> Result<Arg, ArgError> {
   }
 }
 
+fn better_alternative_exists(position: Position, of: &TypePath) -> bool {
+  // If this type will parse without #[serde]/#[to_v8]/#[from_v8] (or with #[string]), it is illegal to use this type
+  // with #[serde]/#[to_v8]/#[from_v8]
+  if parse_type_path(position, Attributes::default(), TypePathContext::None, of)
+    .is_ok()
+  {
+    return true;
+  }
+  // If this type will parse without #[serde] (or with #[string]), it is illegal to use this type with #[serde]
+  if parse_type_path(position, Attributes::string(), TypePathContext::None, of)
+    .is_ok()
+  {
+    return true;
+  }
+
+  // Denylist of serde_v8 types with better alternatives
+  let ty = of.into_token_stream();
+  if let Ok(res) = std::panic::catch_unwind(|| {
+    rules!(ty => {
+      ( Value $( < $_lifetime:lifetime $(,)? >)? ) => true,
```

Comment:
It would be nice to keep the deprecation error for `serde_json::Value`, which I believe this also catches.